src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerLoadImage.groovy

```diff
@@ -0,0 +1,15 @@
+package com.bmuschko.gradle.docker.tasks.image
+
+import com.bmuschko.gradle.docker.tasks.AbstractDockerRemoteApiTask
+import org.gradle.api.tasks.Input
+
+class DockerLoadImage extends AbstractDockerRemoteApiTask {
+
+    @Input
+    InputStream imageStream
```

Comment:
Think I'm leaning against this. I _think_ it's more natural, at least in the gradle world, to just pass in a `File` instance for things of this nature. However I can see the usefulness of passing in an `InputStream` as that can be very flexible. Hmmm ... what about turning this into a `def` which could than allow for the user to pass in either a `File` or `InputStream` instance and we simply check at runtime what we have and proceed accordingly?

Reply:
In Groovy you can easily create `InputStream` from `File`: 
````groovy
new File().newInputStream()
````
so I wouldn't change that.

Reply:
Fair enough